tests/test_sinogram.py

```diff
@@ -135,4 +139,100 @@ def test_multidim(num_ang):
     _nrms = np.sqrt(np.mean((s - reference_sinograms) ** 2, axis=-1)) / np.linalg.norm(
         reference_sinograms, axis=-1
     )
-    np.testing.assert_array_less(_nrms, SK_TOL, "Error in image projections.")
+    np.testing.assert_array_less(
+        _nrms, SK_TOL_FORWARDPROJECT, err_msg="Error in image projections."
+    )
+
+
+def test_backproject_single(masked_image, num_ang):
+    """
+    Test Sinogram.backproject on a single stack of line projections (sinograms).
+
+    This test compares the reconstructed image from the `backproject` method to
+    the skimage method `iradon.`
+    """
+    angles = np.linspace(0, 360, num_ang, endpoint=False)
+    rads = angles / 180 * np.pi
+    sinogram = masked_image.project(rads)
+    sinogram_np = sinogram.asnumpy()
+    back_project = sinogram.backproject(rads)
+
+    assert masked_image.shape == back_project.shape, "The shape must be the same."
+
+    # generate circular mask w/ radius 1 to reconstructed image
+    # aim to remove discrepencies for the edges of the image
+    g = grid_2d(back_project.resolution, normalized=True, shifted=True)
+    mask = g["r"] < 0.99
+    our_back_project = back_project.asnumpy()[0] * mask
+
+    # generating sci-kit image backproject method w/ no filter
+    sk_image_iradon = iradon(sinogram_np[0].T, theta=-angles, filter_name=None) * mask
+
+    # we apply a normalized root mean square error on the images to find relative error to range of ref. image
+    nrmse = np.sqrt(np.mean((our_back_project - sk_image_iradon) ** 2)) / (
+        np.max(sk_image_iradon) - np.min(sk_image_iradon)
+    )
```

Comment:
I notice we're using a different normalization factor here compared to the `project` tests. Do we get the same tolerance if we use this `max - min` normalization there? If so, maybe we can just use one `SK_TOL`.

Reply:
I don't think the normalizations are the same.

<img width="1157" alt="Screenshot 2024-08-15 at 10 39 47 AM" src="https://github.com/user-attachments/assets/4ea382b6-c3c9-43fd-acad-42e0cba1ff10">
<img width="776" alt="Screenshot 2024-08-15 at 10 39 55 AM" src="https://github.com/user-attachments/assets/74b6c569-e0d2-4cd0-bdb9-166783c918fc">

Reply:
I know they're not the same. I was just wondering if the two different methods of normalization accounted for the two different tolerance levels. It appears not to be the case though.